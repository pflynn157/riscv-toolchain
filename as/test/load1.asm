lw x1, 20(x2)
lb x1, 20(x3)
lh x2, 20(x4)
lbu x3, 50(x6)
lhu x4, 60(x8)
