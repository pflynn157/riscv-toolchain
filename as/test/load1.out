01412083
01418083
01421103
03234183
03c45203
